icons/line.svg

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" [
+  <!ENTITY fill_color "#808080">
+  <!ENTITY stroke_color "#FFFFFF">
+]>
+<svg
+   xmlns="http://www.w3.org/2000/svg"
+   version="1.1"
+   width="62"
+   height="60"
+   viewBox="0 0 62 60"
+   id="svg2"
+   xml:space="preserve"><g
+     transform="translate(60.127119,11.292373)"
+     id="activity-browse"
+     style="display:block">
+	
+	<g
+   transform="translate(-128.87712,-22.838983)"
+   id="g7"
+   style="display:inline">
+		
+		
+		
+		
+		
+		
+	<g
+   transform="matrix(0.10822504,0,0,0.09945444,61.358446,34.085169)"
+   id="g6167"><path
+     d="m 594.07188,325.15231 0,-360.47952 L 434.51609,52.953494 274.96046,-175.105 115.4045,109.35506 l 0,215.79736 z"
+     id="rect3757"
+     style="color:#000000;fill:&fill_color;;fill-opacity:1;stroke:&stroke_color;;stroke-width:33.73588181;stroke-linecap:square;stroke-linejoin:miter;stroke-miterlimit:4;stroke-opacity:1;stroke-dasharray:none;stroke-dashoffset:0;marker:none;visibility:visible;display:inline;overflow:visible;enable-background:accumulate" /></g></g>
+</g></svg>
```

Comment:
Missing end of line marker, also in Chart activity.

Reply:
I ran ` sed -i '$a\' line.svg `
If I get something like this in git diff, does it mean I have added eol marker?
```
-}
\ No newline at end of file
+}
```

Reply:
GitHub shows in this discussion last line of the diff a red circle with a slash, and a red end of line mark.

You can verify with most text files by using an editor or program that will display the bytes.

Like `hexdump -C file`.  The line.svg file in Chart activity doesn't have the newline at end of file.  You can add one by typing `echo >> file`.

```log
00000450  2d 62 61 63 6b 67 72 6f  75 6e 64 3a 61 63 63 75  |-background:accu|
00000460  6d 75 6c 61 74 65 22 20  2f 3e 3c 2f 67 3e 3c 2f  |mulate" /></g></|
00000470  67 3e 0a 3c 2f 67 3e 3c  2f 73 76 67 3e           |g>.</g></svg>|
```

after

```log
00000450  2d 62 61 63 6b 67 72 6f  75 6e 64 3a 61 63 63 75  |-background:accu|
00000460  6d 75 6c 61 74 65 22 20  2f 3e 3c 2f 67 3e 3c 2f  |mulate" /></g></|
00000470  67 3e 0a 3c 2f 67 3e 3c  2f 73 76 67 3e 0a        |g>.</g></svg>.|
```

Missing newline at end of file makes life difficult for people who use command line shells.  When you use `cat` or most other programs to display a file, the shell prompt is issued on the same line as the last line of the file.  Then command recall and editing might not work.  It's amazing how such a little thing has been with us since the 1980s.  :grin:

Reply:
(and if you copied the file from Chart, it's not your fault!)